SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the latest release.
+
+## Reporting a Vulnerability
+
+If you have discovered a security vulnerability in this project, please report it privately. **Do not disclose it as a public issue.** This gives us time to work with you to evaluate and fix the issue before public exposure, reducing the chance that the exploit will be used before a patch is released.
+
+Please disclose it privately via email to security@sourcegraph.com. We will work with you to understand and resolve the issue promptly.
```

Comment:
@willdollman should we mention that we have a bug bounty program and will pay a bounty for serious issues?